resources/js/processes/scripts/components/CreateScriptModal.vue

```diff
@@ -9,22 +9,19 @@
     >
       <i class="fas fa-plus" /> {{ $t("Script") }}
     </b-button>
-    <modal
+    <b-modal
```

Comment:
don't change this tag

Reply:
without the tag it wouldn't let me change the footer